conda/cmake-package-test/CMakeLists.txt

```diff
@@ -18,10 +18,10 @@ if(NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")
   )
   file(
     DOWNLOAD
-    "https://raw.githubusercontent.com/conan-io/cmake-conan/v0.16.1/conan.cmake"
+    "https://raw.githubusercontent.com/conan-io/cmake-conan/v0.17.0/conan.cmake"
```

Comment:
This URL 404's, I think you want `https://raw.githubusercontent.com/conan-io/cmake-conan/0.17.0/conan.cmake` (note: no `v`)